src/gui/shell.cpp

```diff
@@ -87,6 +87,8 @@ QString Shell::fontDesc()
 		fdesc += ":b";
 	} else if (font().weight() == QFont::Light) {
 		fdesc += ":l";
+	} else if (font().weight() == QFont::DemiBold) {
+	        fdesc += ":sb";
```

Comment:
mixed tabs and spaces

Reply:
I noticed that.  Why do you even have tabs?

Reply:
Because tabs are better ;)